resources/definitions/kosher_duplication.def.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 2,
+    "name": "Kosher_Duplication",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Venkat Kamesh",
+        "manufacturer": "Sri Vignan Technologies",
+        "weight": 3,
+        "file_formats": "text/x-gcode",
+        "platform": "ultimaker3_platform.obj",
+        "platform_texture": "koshertext.png",
+        "platform_offset": [52.5, 0, 0],
+        "has_material": true,
+        "has_variants": true,
+        "preferred_variant_name": "0.4 mm",
+        "machine_extruder_trains":
+        {
+            "0": "kosher_dm_extruder"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Kosher_Duplication" },
+        "machine_width": { "default_value": 100 },
+        "machine_height": { "default_value": 300 },
+        "machine_depth": { "default_value": 220 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_acceleration": {"default_value": 3000},
+        "machine_max_feedrate_x": { "value": 300 },
+        "machine_max_feedrate_y": { "value": 300 },
+        "machine_max_feedrate_z": { "value": 40 },
+        "machine_max_feedrate_e": { "value": 150 },
+        "machine_use_extruder_offset_to_offset_coords": {"default_value": false},
+        "machine_center_is_zero": {"default_value": false},
+        "gantry_height": {"value": "48"},
+        "line_width": {"value": "machine_nozzle_size"},
+        "speed_travel": {"maximum_value": "300", "value": "200"},
+        "optimize_wall_printing_order": { "value": "True" },
+        "material_diameter": { "default_value": 1.75},
+        "retraction_amount": {"default_value": 6.5},
+        "retraction_speed": { "default_value": 30},
+        
+		"adhesion_type": { "default_value": "skirt" },
+        "ironing_enabled":{"label": "Enable Ironing",
```

Comment:
Same as before; Don't override settings with the same value. THis could cause issues in the future.

Reply:
Could you please point it out?

Reply:
You are overriding the label of a setting. If the label of this setting is changed in fdmprinter, this one won't be translated any more. I'd say this is not what you aimed for. There is no reason why you'd change the label of a setting for a specific printer, I'd think.

Overriding the value of a setting with the same value as what it was before can be done fairly harmlessly, but it's also useless since it just ends up on the same setting value then.